torch/nn/_functions/batchnorm.py

```diff
@@ -20,6 +20,7 @@ def forward(self, input, weight=None, bias=None):
         # don't use cuDNN for half inputs because cuDNN requires the weight and
         # bias tensors to be floats, unlike THCUNN which requires half tensors.
         self.use_cudnn = (cudnn.is_acceptable(input) and
+                          cudnn.version() > 5110 and
                           weight is not None and bias is not None and
                           not isinstance(input, torch.cuda.HalfTensor))
```

Comment:
Why? THCUNN with affine transformation and cuDNN should be fully compatible. @eladhoffer had PR #901 that would make batchNorm for inference unnecessary, everyone should use that, it's faster.